Shared/Notifications/UserNotificationManager.swift

```diff
@@ -105,7 +133,7 @@ class UserNotificationManager: NSObject {
                     if note.request.identifier == message.id {
                         identifiers.append(message.id)
                     }
-                    
+
                     if message.context == .timeSensitive {
                         badgeCount -= 1
```

Comment:
I forgot that the installation has a badge count on it... refactor for another time.